plugins/statistics.koplugin/main.lua

```diff
@@ -1074,6 +1074,46 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                         end,
                         separator = true,
                     },
+                    {
+                        text_func = function()
+                            -- @translators %1 is the time in the format 00:00
+                            return T(_("Daily timeline starts at %1"),
+                                string.format("%02d:%02d", self.settings.calendar_day_start_hour or 0,
+                                                           self.settings.calendar_day_start_minute or 0)
+                            )
+                        end,
+                        callback = function(touchmenu_instance)
+                            local DateTimeWidget = require("ui/widget/datetimewidget")
+                            local start_of_day_widget = DateTimeWidget:new{
+                                hour = self.settings.calendar_day_start_hour or 0,
+                                min = self.settings.calendar_day_start_minute or 0,
+                                hour_max = 6,
+                                ok_text = _("Set time"),
+                                title_text = _("Set starting time of a day"),
+                                info_text =_([[
+Set the time when the day timeline should start.
+
+If you read past midnight, and wish this reading to appear with the previous evening's reading, you can set this to 04:00 for example.
```

Comment:
```suggestion
If you read past midnight, and wish this reading session to be considered part of the previous evening, use a value such as 04:00.
```

Reply:
You misinterpreted that, or "reading session" is not clear, or confusing.
May be then:
If you read past midnight, and wish this reading to be displayed on the same screen with your previous evening reading, use a value such as 04:00.

Reply:
I probably did, but "reading" feels ungrammatical in any case.

Reply:
So, may be "reading session(s)" (it feels the second part needs plural): 
If you read past midnight, and wish this reading session to be displayed on the same screen with your previous evening reading sessions, use a value such as 04:00.

Reply:
Wording updated :)